node/node_test.go

```diff
@@ -3795,6 +3797,111 @@ func TestNode_GetHeartbeats(t *testing.T) {
 	})
 }
 
+func TestNode_GetEpochStartDataForEpoch(t *testing.T) {
+	t.Parallel()
+
+	t.Run("epoch 0 - should return genesis", func(t *testing.T) {
+		n, _ := node.NewNode(
```

Comment:
i think we can safely run also the inner tests in paralel

Reply:
right, added